internal/internal_worker.go

```diff
@@ -39,6 +39,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	"go.uber.org/cadence/internal/debug"
```

Comment:
nit: I think this should be in the same imports group as isolationgroup. But I guess we are missing gci integration. I think it might be fixed in a separate PR with CI enforcement.